packages/devtools_app/lib/src/screens/deep_link_validation/deep_link_list_view.dart

```diff
@@ -104,11 +104,39 @@ class _DeepLinkListViewMainPanel extends StatelessWidget {
             return const _ValidatedDeepLinksView();
           case PagePhase.noLinks:
             // TODO(hangyujin): This is just a place holder to add UI.
-            return const Text('Your flutter project has no Links to verify.');
+            return const Center(child: Text('Your flutter project has no Links to verify.'));
+          case PagePhase.analyzeErrorPage:
+            assert(controller.currentAppLinkSettings?.error != null);
+            return Column(
+              children: [
+                const Text(
+                  'Failed to retrieve deep links from the Flutter project. This can be a result of errors in the project.',
+                ),
+                const SizedBox(height: densePadding),
+                const Text(
+                  'Error Message:',
+                ),
+                Expanded(
+                  child: Padding(
+                    padding: const EdgeInsets.only(left: 20),
+                    child: OutlineDecoration(
```

Comment:
I would remove the outline decoration here. I think the error color for the text differentiates this from the text above.

Reply:
I added this to draw the boundary for the scrollview. so that people know this area is scrollable. Do you think border is better? or is there a prefer style I should follow